timezone/src/modules/Timezone.rb

```diff
@@ -1002,8 +1007,16 @@ def Summary
 
     # Checks whether the system has Windows installed
     def system_has_windows?
-      win_partitions = disk_analyzer.windows_partitions
-      !win_partitions.empty?
+      begin
+        win_partitions = disk_analyzer.windows_partitions
+        !win_partitions.empty?
+      rescue NameError => ex
+        # bsc#1058869: Don't enforce y2storage being available
+        log.warn("Caught #{ex}")
+        log.warn("No storage-ng support - not checking for a windows partition")
+        log.warn("Assuming UTC for the hardware clock")
+        false # No windows partition found
+      end
```

Comment:
btw this can be written as

```
def system_has_windows?
  require "y2storage"
  disk_analyzer = Y2Storage::StorageManager.instance.probed_disk_analyzer
  win_partitions = disk_analyzer.windows_partitions
  !win_partitions.empty?
rescue LoadError
  log.warn("No storage-ng support - not checking for a windows partition")
  false
end
```

which quite simplify all these stuff ( disk analyzer method can be removed and similar that require on top )